bin/collator/src/cli.rs

```diff
@@ -53,6 +53,16 @@ pub struct Cli {
     /// Proposer's soft deadline in percents of block size
     #[clap(long, default_value = "50")]
     pub proposer_soft_deadline_percent: u8,
+
+    /// Disable automatic hardware benchmarks.
+    ///
+    /// By default these benchmarks are automatically ran at startup and measure
+    /// the CPU speed, the memory bandwidth and the disk speed.
+    ///
+    /// The results are then printed out in the logs, and also sent as part of
+    /// telemetry, if telemetry is enabled.
+    #[arg(long)]
+    pub no_hardware_benchmarks: bool,
```

Comment:
Q - why do we want to provide an option to disable this?

Result is only a warning.

Reply:
polkadot-parachain has it. Just in case someone doesn't want it, they can disable it